modes/debug_mode/debug_plot_page.py

```diff
@@ -112,12 +114,23 @@ def update(self):
                 text=self.data[id].data[0], text_color=self.colors[i])
 
             # plotting the graph
-            y = self.data[id].data
-            self.ax.plot(y[0: self.current_time] if len(y) > self.current_time else y, color=self.colors[i])
+            transformed_data = self.transform_data_to_time(self.data[id].data)
+            y = np.array(transformed_data)
+            self.ax.plot(y, color=self.colors[i])
+            self.ax.xaxis.set_major_formatter(lambda x, pos: str(int(x / 600 * self.current_time)) + "s")
             i += 1
         self.ax.invert_xaxis()
         for j in range(i, 5):
             self.key_frames[j].name_label.configure(text="")
             self.key_frames[j].unit_label.configure(text="")
             self.key_frames[j].current_value_label.configure(text="")
         self.canvas.draw()
+
+    def transform_data_to_time(self, data: List[float]) -> List[float]:
```

Comment:
python docstring pls

https://github.com/google/styleguide/blob/gh-pages/pyguide.md#383-functions-and-methods